Sources/CodeEditTextView/STTextViewController.swift

```diff
@@ -78,22 +83,19 @@ public class STTextViewController: NSViewController, STTextViewDelegate, ThemeAt
     // MARK: VC Lifecycle
 
     public override func loadView() {
-        let scrollView = STTextView.scrollableTextView()
-        textView = scrollView.documentView as? STTextView
-
-        // By default this is always null but is required for a couple operations
-        // during highlighting so we make a new one manually.
-        textView.textContainer.replaceLayoutManager(NSLayoutManager())
-
+        textView = STTextView()
+        
+        let scrollView = NSScrollView()
         scrollView.translatesAutoresizingMaskIntoConstraints = false
         scrollView.hasVerticalScroller = true
+        scrollView.documentView = textView
 
         rulerView = STLineNumberRulerView(textView: textView, scrollView: scrollView)
         rulerView.backgroundColor = theme.background
         rulerView.textColor = .systemGray
         rulerView.drawSeparator = false
         rulerView.baselineOffset = baselineOffset
-        rulerView.font = NSFont.monospacedDigitSystemFont(ofSize: 9.5, weight: .regular)
+        rulerView.font = NSFont(descriptor: NSFont.monospacedDigitSystemFont(ofSize: 11, weight: .regular).fontDescriptor, textTransform: AffineTransform(scaleByX: 1, byY: 1.1)) ?? .monospacedDigitSystemFont(ofSize: 11, weight: .regular)
```

Comment:
Why do we need to scale the font on the y axis?

Reply:
That was mistakenly committed. in the process of cleaning this draft up- woops

Reply:
@lukepistrol , addressed. That code was a misguided attempt at mimicking the Xcode ruler font without having to rely on SF variable font support (which is only available in OS 13+).

Reply:
@ben-p-commits As mentioned, we are relying on 13 for other things so it is safe to use variable fonts.